mesonbuild/mesonmain.py

```diff
@@ -40,6 +40,8 @@ def create_parser():
                    help='Special wrap mode to use')
     p.add_argument('--profile-self', action='store_true', dest='profile',
                    help=argparse.SUPPRESS)
+    p.add_argument('--reconfigure', action='store_true',
+                   help='Reconfigure the project using the same options.')
```

Comment:
This is a new feature, so it needs @jpakkane's approval.

Reply:
Right, I could split my PR before that commit so the rest could still go. Is there a way in github to tell a PR depends on another PR ?